### PR TITLE
Allows to retrieve client’s paymethods

### PR DESCRIPTION
**Pull request by hovancik**
Hi. I'm trying this gem for a project I'm working on so I'm adding some missing stuff. Could you please review the code? 

related docs: http://docs.payu21.apiary.io/#reference/api-endpoints/paymethods-api-endpoint/retrieve-paymethods-and-tokens

### CODE REVIEW

File: lib/payu_api/responses/paymethods_response.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This methods allows to retrieve paymethods which are enabled for pos.
+# For merchants with token payments enabled it also retrieves user payments tokens.
+# Token payments requires OAuth token with grant type trusted_merchant.
+module PayuAPI
+  class PaymethodsResponse < Response
+    def response
```
Comment (hovancik) [inline lib/payu_api/responses/paymethods_response.rb]: Nope. I'll remove it.

File: lib/payu_api/responses/paymethods_response.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      body
+    end
+
+    def card_tokens
+      body[:cardTokens]
```
Comment (hovancik) [inline lib/payu_api/responses/paymethods_response.rb]: Makes sense. Will do.

**Comment by hovancik** [comment]
I've made the changes. Not sure about the tests, thought.

File: test/payu_apu/paymethods_response_test.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class PayuAPI::ResponseTest < Minitest::Test
```
Comment (hrom512) [inline test/payu_apu/paymethods_response_test.rb]: Class name is incorrect. Right is `PayuAPI::PaymethodsResponseTest`.

Comment (hrom512) [inline test/payu_apu/paymethods_response_test.rb]: Also you need to move file to `test/payu_apu/response/`

Comment (hovancik) [inline test/payu_apu/paymethods_response_test.rb]: Done. Why is it `payu_apu` btw? Typo?

Comment (hrom512) [inline test/payu_apu/paymethods_response_test.rb]: Yes, this is typo)

Comment (hovancik) [inline test/payu_apu/paymethods_response_test.rb]: Should I fix it now, or in another PR?

Comment (hrom512) [inline test/payu_apu/paymethods_response_test.rb]: I'll fix this in another PR.

Comment (hovancik) [inline test/payu_apu/paymethods_response_test.rb]: 👍

**Pull request merged**